app/src/main/java/com/opsat7373/moviestars/adapters/MovieItemAdapter.kt

```diff
@@ -0,0 +1,41 @@
+package com.opsat7373.moviestars.adapters
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.opsat7373.moviestars.R
+import com.opsat7373.moviestars.databinding.MovieItemBinding
+import com.opsat7373.moviestars.data.model.Movie
+import java.util.*
+
+class MovieItemAdapter(moviesList : List<Movie?>?) : RecyclerView.Adapter<MovieItemAdapter.MovieItemViewHolder>() {
```

Comment:
looks like passing data to constructor is useless, as you update it with setList() method